lib/yamel.ex

```diff
@@ -8,6 +8,20 @@ defmodule Yamel do
   @type keys :: :atoms | :strings
   @type decode_opt :: {:keys, keys}
 
+  @doc ~S"""
+  Decode a YAML string into it respective data type in elixir, throwing an exception if the
+  string can not be encoded
+
+  ## Examples
+
+      iex> Yamel.decode!("
+      ...>name: Jane Doe
+      ...>job: Developer
+      ...>skill: Elite
+      ...>employed: True")
```

Comment:
@uduDudu could you remove all those `...>` so the output can match? If you copy and past this as is (which most of the times is the purpose of this example), it will yield another output than the one below.

Reply:
When i try to do it without the `...>` the doctest throws an error of missing terminator `"`, and when i paste the YAML in the iex it turns like the example, that why i put like this, can you please give a example of how is suppose to be? (Also i fixed the number of the issue in the PR, didnt see that referenced the wrong one, sorry)

Reply:
@uduDudu if you copy the following snippet from the documentation and paste it on the repl `iex -S mix`:

```elixir
Yamel.decode!("
      ...>name: Jane Doe
      ...>job: Developer
      ...>skill: Elite
      ...>employed: True")

=> %{
  "...>employed" => true,
  "...>job" => "Developer",
  "...>name" => "Jane Doe",
  "...>skill" => "Elite"
}
```

which is correct, but it's kinda odd.

Try to use `~S"""..."""` instead of `"..."` as the argument of the `Yamel.decode!/1` function.

Reply:
@GPrimola im still having some errors, maybe i will change the examples to one line just to be more simple, if i dont find a way to solve it

Reply:
@uduDudu I've cloned your repo to see what's going on. Thing is:

```
If instead you have an `iex>` expression that spans over multiple lines, please make sure that each line after the first one begins with `...>`.
```

This is one of the errors that appear when testing with this doctest. I don't know if you could understand it, but what it says is that `...>` continues the `iex>`. So, with that in mind, my suggestion is:

```elixir
iex> Yamel.decode!("
...>     name: Jane Doe
...>     job: Developer
...>     skill: Elite
...>     employed: True")

```

This is way more readable than it was before. WDYT?

Reply:
@GPrimola Yeah i did it, much better to read